### ISSUE TITLE
login after successful login

### ISSUE THREAD

**Issue by Sravs66** [jcubic/jquery.terminal#502]
Hi jCubic,
After successful login into the terminal, if user types login its coming out and prompting for login again...but before that i need to clear the terminal so how can i achieve that..?

![Login Issue](https://user-images.githubusercontent.com/50399157/59189552-f5b34580-8b97-11e9-969c-a379fb0eeeb1.png)

**Comment by jcubic** [comment]
You can use universal onCommand event:
```javascript
var term = $('body').terminal({
    x: function() {
        this.echo('x');
    }
}, {
    onAfterCommand: function(command) {
        // this is called for every command except for those that didn't get executed
        // if onBeforeCommand callback return false it will not log that comment it will be ignored
        if (command.match(/^\s*login\s*$/)) {
            this.clear();
        }
    },
    name: 'login_q',
    login: function(login, pass, callback) {
        callback(login + pass == 'foobar' ? 'TOKEN': null);
    }
});
```

**Comment by jcubic** [comment]
I need to update the code to have onBefore and onAfterLogin to be called when you call login as command.

**Comment by Sravs66** [comment]
Sure...thanks

**Comment by Sravs66** [comment]
Hi jCubic, 
I observed one problem with this approach...this onAfterCommand is getting called for the first time only...i mean to say first time if i call login on already logged in session its fine working. But without closing the window if again i'm logging in and calling login, onAfterCommand itself is not getting called...!

**Comment by jcubic** [comment]
Can you show exact steps? `onAfterCommand` should always be called. Short example would be helpful.

**Comment by Sravs66** [comment]
Actually i'm using this plugin inside of ExtJs window,

1. I opened extjs window where, inside i'm setting this terminal to one Exxtjs component.
2.  I created a session with login details.
3. Executed some commands
4. Then typed login(Worked perfectly)(Here, i'm doing some things related to session terminating)
5. Now again i logged in
6. Executed some commands.
7. Typed login but the terminal is not cleared...not even debugger went to onAfterCommand method.
